RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/Transformer_ES_6_8_to_OS_2_11.java

```diff
@@ -27,10 +27,13 @@ public Transformer_ES_6_8_to_OS_2_11(int awarenessAttributeDimensionality, Metad
         this.awarenessAttributeDimensionality = awarenessAttributeDimensionality;
         this.indexTransformations = List.of(new IndexMappingTypeRemoval(
                 params.getMultiTypeResolutionBehavior()
-        ));
-        this.indexTemplateTransformations = List.of(new IndexMappingTypeRemoval(
+            )
+        );
+        this.indexTemplateTransformations = List.of(
+            new IndexMappingTypeRemoval(
                 params.getMultiTypeResolutionBehavior()
```

Comment:
This seems fine, but I'm not sure I understand why it was possible to skip this before -- was it just not handling multitypes for index templates?